src/pages/index.page.tsx

```diff
@@ -19,6 +20,8 @@ function Landing() {
 
       <TrendingSection />
 
+      <TagsSpotlightSection />
```

Comment:
**THIS IS AN EXAMPLE COMPONENT**

it's functionally fine, but we shouldn't ship this as is since it looks kinda meh

Reply:
jk removed this